infrastructure/task/scripts/redeploy-failed-deployments.sh

```diff
@@ -25,6 +25,9 @@ for deployment in $FAILED_PRODUCTION_DEPLOYMENTS; do
   if [[ $deployment =~ "dpl-cms" ]]; then
     continue
   fi
+  if [[ $deployment =~ "dpl-bnf" ]]; then
```

Comment:
The changes in this file is what triggered me to make the controller - leave it in here, so we can run it manually, should we need to